catalog/imaging/transformer/catalog-transformer-nitf/src/test/java/ddf/catalog/transformer/nitf/TestNitfInputTransformer.java

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.nitf;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.awt.Toolkit;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mockito.Matchers;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.CatalogFramework;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.dynamic.api.MetacardFactory;
+import ddf.catalog.data.dynamic.impl.MetacardFactoryImpl;
+import ddf.catalog.data.dynamic.registry.MetacardTypeReader;
+import ddf.catalog.federation.FederationException;
+import ddf.catalog.operation.QueryRequest;
+import ddf.catalog.operation.impl.QueryResponseImpl;
+import ddf.catalog.source.SourceUnavailableException;
+import ddf.catalog.source.UnsupportedQueryException;
+import ddf.catalog.transform.CatalogTransformerException;
+
+public class TestNitfInputTransformer {
+    private static final Logger LOGGER = LoggerFactory.getLogger(TestNitfInputTransformer.class);
+
+    private static final String BE_NUM_NITF = "/WithBE.ntf";
+
+    private static final String TRE_NITF = "/i_3128b.ntf";
+
+    private static final String GEO_NITF = "/i_3001a.ntf";
+
+    private static MetacardFactory metacardFactory = null;
+
+    private NitfInputTransformer transformer = null;
+
+    @BeforeClass
+    public static void setupBeforeClass() throws Exception {
+        MetacardFactory mf = new MetacardFactoryImpl();
```

Comment:
This local `MetacardFactory` isn't necessary - you could just work with `metacardFactory`.